templates/geosafe/analysis/create.html

```diff
@@ -520,7 +520,13 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
             });
 
             {# Zoom first to global world view #}
-            zoom_to_box(map, [0, 45, 0, -45]);
+            {%  if analysis and analysis.user_extent %}
+                user_extent_string = "{{ analysis.user_extent }}";
+                extent_after_run = user_extent_string.split(",");
+                zoom_to_box(map, extent_after_run);
```

Comment:
I think this is not going to work because after the split, the list contains string, not float.
That's why I suggest you use: `"[" + {{ analysis.user_extent }} + "]"`

To illustrate my comment, let's suppose in python:

```
analysis.user_extent = "106,-6,107,-9"
```
Your code will look like this after template rendering:

```
user_extent_string = "106,-6,107,-9";
extent_after_run = user_extent_string.split(",");
zoom_to_box(map, extent_after_run);
```

This will not work, because `extent_after_run` will contain: `["106","-6","107","-9"]`. The list contains strings, meanwhile zoom_to_bbox expects list of float.

If you use my suggestion:

```
extent_after_run = [ {{ analysis.user_extent }} ];
```

It will be like this after template render:

```
extent_after_run = [ 106,-6,107,-9 ];
```

Try this first.